stylesheets/hacks.css

```diff
@@ -4,6 +4,13 @@
  * everything here.
  */
 
+/* fix active-zone-selector being to far from the logo. The selector
+ * still has bad css when using mobile or small screen view.
+ */
+.active-zone-selector__singlezone {
+    margin-left: auto !important;
```

Comment:
Are you using `!important` because you need to override inline styles?  Does this change apply to cPanel and Magento as well?  Would it make sense to move it inline in `ActiveZoneSelector.js` in `cloudflare-plugin-frontend`?